<ui version="4.0" >
 <class>GridSpaceDialog</class>
 <widget class="QDialog" name="GridSpaceDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Define the Grid Space</string>
  </property>
  <widget class="QDialogButtonBox" name="okCancelButtons" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>360</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>100</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title" >
    <string>Define the experiment space in meters</string>
   </property>
   <widget class="QLineEdit" name="_xSpacing" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>205</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Width (X) in multiples of 1.6 meters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>207</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Height (Y) in multiples of 1.6 meters</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_ySpacing" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>150</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Depth (Z) total meters</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_depth" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>170</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>20</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Name for this experiment</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="spaceName" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>50</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>accepted()</signal>
   <receiver>GridSpaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>rejected()</signal>
   <receiver>GridSpaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
